### ISSUE TITLE
Too frequent Warning: Your connection is not stable. Please check your connection 

### ISSUE THREAD

**Issue by mekya** [ant-media/conference-call-application#64]
While I'm in the call, I receive this warning frequently.  
`Your connection is not stable. Please check your connection`

On the other hand, my video call experience was good. 

<img width="1481" alt="Screenshot 2023-03-27 at 15 16 20" src="https://user-images.githubusercontent.com/3456251/227943249-63fb0f14-a0dc-4c5a-aca9-fb383414a867.png">

This is the debugme output 
```
Client Debug Info
Events:
[{"track":"ARDAMSaaudioTrack0","event":"added"},{"track":"ARDAMSaaudioTrack1","event":"added"},{"track":"ARDAMSaaudioTrack2","event":"added"},{"track":"ARDAMSvvideoTrack0","event":"added"},{"track":"ARDAMSvvideoTrack0","event":"removed"},{"track":"ARDAMSvvideoTrack1","event":"added"},{"track":"ARDAMSvvideoTrack2","event":"added"},{"track":"ARDAMSvvideoTrack3","event":"added"},{"track":"ARDAMSvvideoTrack4","event":"added"},{"track":"ARDAMSvvideoTrack5","event":"added"},{"track":"ARDAMSvvideoTrack6","event":"added"},{"track":"ARDAMSvvideoTrack1","event":"removed"},{"track":"ARDAMSvvideoTrack6","event":"removed"}]
Participants (1):
[{"id":"Murat_qOia6kqp2e","videoLabel":"videoTrack5","track":{},"streamId":"tech","isCameraOn":true,"name":"Murat","oldId":"Tim_acsDgGjabR"}]
----------------------
Server Debug Info
Self Track Id:Oguz_JsivAd7cp5

Video Track Map:
Murat_qOia6kqp2e:true
Oguz_JsivAd7cp5:false
Alper_6AeX92DgkG:true

Video Track Assignments (4/6)
{label:videoTrack2,assigned stream id:,reserved:false,prevAssignedStreamId:null,lastSentTime:1679919209012,ts:2362800000000,offset:-143080000000,keyFrameSent:true,}
{label:videoTrack3,assigned stream id:,reserved:false,prevAssignedStreamId:null,lastSentTime:1679918700054,ts:2448200000000,offset:-1152040000000,keyFrameSent:true,}
{label:videoTrack4,assigned stream id:,reserved:false,prevAssignedStreamId:null,lastSentTime:1679918754990,ts:1869710000000,offset:-372790000000,keyFrameSent:true,}
{label:videoTrack5,assigned stream id:Murat_qOia6kqp2e,reserved:true,prevAssignedStreamId:null,lastSentTime:1679919385211,ts:2696060000000,offset:-77440000000,keyFrameSent:true,}

Audio Track Assignments (3/3)
{label:audioTrack0,assigned stream id:Murat_qOia6kqp2e,currentLevel:7,score:1894}
{label:audioTrack1,assigned stream id:burak_PRqmDSxvXB,currentLevel:0,score:467}
{label:audioTrack2,assigned stream id:yash_XeaNmjMHNV,currentLevel:0,score:4036}
```

**Comment by USAMAWIZARD** [comment]
not reproducible at my end.
This is how I tested :
I tried joining the meeting then disconnecting the internet and again re connected the internet.
the stream gets reconnected but I did not get too frequent messages.

**Comment by mustafaboleken** [comment]
I just merged #98 pr but it's a workaround. We should find the real reason. After reconnection issues solved completely in conference call application, we can visit this issue again.
